src/task_scl_eff_pot_hab.py

```diff
@@ -44,8 +49,8 @@ class SCLPolygons(SCLTask):
         },
         "density": {
             "ee_type": SCLTask.FEATURECOLLECTION,
-            "ee_path": "projects/SCL/v1/Panthera_tigris/source/density/tiger_obs_density",
-            "static": True,
+            "ee_path": "projects/SCL/v1/Panthera_tigris/biome_density",
+            "maxage": 10,
```

Comment:
No `maxage` checking for featurecollections (yet) so I'd remove this

Reply:
done